pkg/converter/namespace_converter_test.go

```diff
@@ -0,0 +1,99 @@
+package converter_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/projectcalico/k8s-policy/pkg/converter"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sapi "k8s.io/client-go/pkg/api/v1"
+)
+
+var _ = Describe("NamespaceConverter", func() {
+	nsConverter := converter.NewNamespaceConverter()
+	Context("should parse a Namespace to a Profile", func() {
```

Comment:
I think this is missing a test for `GetKey` (as well as the other conversion files).  Should be a quick one to add.